python/VERSION

```diff
@@ -0,0 +1 @@
+0.9.0
```

Comment:
ide complaining about no line at eof

Reply:
also do you think we could name this a little differently ? like `python_sdk/VERSION` ? python could be confusing